tests/test_container_native.py

```
@@ -0,0 +1,472 @@
+    @pytest.mark.asyncio
+    async def test_push_file_accepts_allowed_root(self, backend, tmp_path, monkeypatch):
+        """push_file must accept paths within _ALLOWED_WRITE_ROOTS."""
+        import tinyagentos.containers.native as _mod
+        # Override allowed roots to include our tmp_path
+        monkeypatch.setattr(
+            _mod, "_ALLOWED_WRITE_ROOTS", (str(tmp_path) + "/",),
+        )
+        src = tmp_path / "source.txt"
+        src.write_text("safe data")
+        dest = tmp_path / "agent" / "config.yaml"
+        code, output = await backend.push_file("ignored", str(src), str(dest))
```

Ruff (0.15.20)

[warning] Unpacked variable `output` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)